delivery_easypost_oca/models/delivery_carrier.py

```diff
@@ -0,0 +1,443 @@
+from datetime import datetime
+
+from markupsafe import Markup
+
+from odoo import _, fields, models
```

Comment:
```suggestion
from odoo import fields, models
```
The recommended way to use translation in odoo 18 is `self.env._`